### ISSUE TITLE
Context & Project choices should be case-insensitive

### ISSUE THREAD

**Issue by ginatrapani** [todotxt/todo.txt-ios#114]
Right now if I have an `@Phone` and an `@phone` context in my todo.txt, they get listed individually in the context picker. The picker should collapse contexts into one where the only difference is case.

**Comment by marsepu** [comment]
a simple way to fix this is to only add the lowercase version of each context to the picker. Would this be an OK fix for you? It's a simple patch that I have already banged out if you want it.

**Comment by ginatrapani** [comment]
Yes, please send on the pull request so I can test out the fix. Thank you!
On Jul 6, 2012 9:09 PM, "Mario Sepulveda" <
reply@reply.github.com>
wrote:

**Comment by chuckbjones** [comment]
@marsepu, can you send a pull request for this one as well? Thanks

**Comment by marsepu** [comment]
Charles,

I sent a pull request for this over a week ago.

-Mario

**Comment by chuckbjones** [comment]
@ginatrapani This is another one that looks like it was never applied.

**Comment by ginatrapani** [comment]
When I went to apply this locally, there was a conflict:

```
<<<<<<< HEAD
        [projects addObject:[[inputText substringWithRange:[match rangeAtIndex:1]] stringByTrimmingCharactersInSet: [NSCharacterSet whitespaceCharacterSet]]];
=======
    NSString* theProjectInLowercase = [[inputText substringWithRange:[match rangeAtIndex:1]] lowercaseString];
        [projects addObject:theProjectInLowercase];
>>>>>>> d2c10de... Fixed issue #114  - Context & Project choices should be case-insensitive
```

I want to get the resolution right. I shouldn't delete stringByTrimmingCharactersInSet: [NSCharacterSet whitespaceCharacterSet]]] but I'm not confident about the right way to merge these changes.

**Comment by chuckbjones** [comment]
I'm not sure why it's trimming whitespace in the first place, but I just sent a new PR that maintains it.
